modules/creating-serverless-apps-kn.adoc

```diff
@@ -32,21 +32,21 @@ Where:
 .Example command
 [source,terminal]
 ----
-$ kn service create event-display \
-    --image quay.io/openshift-knative/knative-eventing-sources-event-display:latest
+$ kn service create showcase \
+    --image quay.io/openshift-knative/showcase
 ----
 +
 .Example output
 [source,terminal]
 ----
-Creating service 'event-display' in namespace 'default':
+Creating service 'showcase' in namespace 'default':
 
   0.271s The Route is still working to reflect the latest desired specification.
-  0.580s Configuration "event-display" is waiting for a Revision to become ready.
+  0.580s Configuration "showcase" is waiting for a Revision to become ready.
```

Comment:
Should route and revision have initial caps?

Reply:
This is an output of `kn` command, so we can't change it here.

We can propose to change the `kn` command itself, in [its repo](https://github.com/knative/client). I think the Knative contributors wrote those with caps, as being a Knative's proper names.

Reply:
No worries. Thanks